modules/auth/user_form.go

```diff
@@ -84,6 +87,31 @@ func (f *RegisterForm) Validate(ctx *macaron.Context, errs binding.Errors) bindi
 	return validate(errs, ctx.Data, f, ctx.Locale)
 }
 
+// IsEmaildomainwhitelisted validates that the email address
+// provided by the user matches what has been configured .
+// If the domain whitelist from the config is empty, it marks the
+// email as whitelisted
+func (f RegisterForm) IsEmaildomainwhitelisted() bool {
```

Comment:
Please fix function name casing `IsEmailDomainWhitelisted`

Reply:
This was what I originally had but you requested it to be changed in  https://github.com/go-gitea/gitea/pull/5157#discussion_r228081394 ... Would fix